### PR TITLE
Add category field to recipes

### LINKED ISSUE #92: Add category to recipes
Once we add category, we should order the recipes by category first and then by difficulty.

### PR DESCRIPTION
**Pull request by marco-c**
Fixes #92.

Suggestions welcome on how to categorize the recipes where I put "?".

### CODE REVIEW

File: message-relay/README.md [hunk 1]
```diff
@@ -12,3 +12,6 @@ The `postMessage` API is brilliant for passing messages between windows and `ifr
 
 - postMessage API
 - Service worker registration
+
+## Category
+?
```
Comment (darkwing) [inline message-relay/README.md]: How about "Messaging"?

**Comment by brendandahl** [comment]
We could put all the ones with only one category in an other misc category for now.

**Comment by marco-c** [comment]
Sounds good to me, done in 9ad0530.

**Comment by darkwing** [comment]
R+!

**Pull request merged**